testdata/bootstrap/client.yaml.tmpl

```diff
@@ -40,6 +40,13 @@ static_resources:
     connect_timeout: 1s
     type: STATIC
     http2_protocol_options: {}
+    metadata:
```

Comment:
Can you guard this for now with `{{ if }}`, so we make sure that MX is actually exercised? It's hard to tell if it always falls back.

Reply:
I think you mean the endpoint labels? I added a wrapper to the endpoint.

Reply:
Can you change the name then of the cluster? I think you imply that the cluster metadata is mandatory, which is fine, but need to be clear.

Reply:
Done.